### PR TITLE
[xc-admin] permission accounts

### PR DESCRIPTION
**Pull request by cctdaniel**
- add update permissions tab

### CODE REVIEW

File: governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx [hunk 1]
```diff
@@ -1,13 +1,99 @@
...
+                    }
+                  >
+                    {tab[1].title}
+                  </Tab>
+                ))}
+              </Tab.List>
+            </Tab.Group>
+          </div>
+        </div>
+        {currentTabIndex !== -1 &&
```
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx:89]: When can it be equal to -1

File: governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx [hunk 2]
```diff
@@ -1,13 +1,99 @@
...
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
 import Layout from '../components/layout/Layout'
-import MinPublishers from '../components/MinPublishers'
+import MinPublishers from '../components/tabs/MinPublishers'
+import UpdatePermissions from '../components/tabs/UpdatePermissions'
 import { PythContextProvider } from '../contexts/PythContext'
+import { classNames } from '../utils/classNames'
+
+const tabInfo = {
```
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx:11]: capitalize this so it's easier to know it's a const

File: governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx (hunk 1 above)
Comment (cctdaniel) [reply governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx:89]: when we visit the site without passing in query string `tab` at line 51

File: governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx [hunk 3]
```diff
@@ -1,13 +1,99 @@
...
+    title: 'Update Permissions',
+    description: 'Update the permissions of the program.',
+    queryString: 'update-permissions',
+  },
+}
+
+const DEFAULT_TAB = 'min-publishers'
 
 const Home: NextPage = () => {
+  const [currentTabIndex, setCurrentTabIndex] = useState(0)
```
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx:28]: Does this need to be a number, wouldn't it make more sense as a "updatePermissions" | "setMinPub" or a string

**Review by guibescos — APPROVED** [review]
Logic of tabs seems complicated, but there might be a reason why that I don't know

File: governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx (hunk 3 above)
Comment (cctdaniel) [reply governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx:28]: yes -- we are using Tabs from @headless-ui and they require index to be passed in for their onChange handler

File: governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx (hunk 1 above)
Comment (cctdaniel) [reply governance/xc-admin/packages/xc-admin-frontend/pages/index.tsx]: good catch, this was for a previous bug that has been fixed, will remove

**Pull request merged**